### ISSUE TITLE
Dark mode not working when high contrast mode is turned on.

### ISSUE THREAD

**Issue by raymond-ntt** [desktop/desktop#13846]
### Describe the bug

Whenever Windows high contrast mode is turned on, dark theme on GitHub Desktop will not work and is mostly white.
Whenever Windows high contrast mode is turned off, dark theme on GitHub Desktop works. 

I have tried a fresh install but it isn't working as well.

### Version & OS

Running on Windows 10 x64
GitHub Desktop 2.9.6

### Steps to reproduce the behavior

1. Turn on Windows High Contrast Mode (Ctrl + Alt + Delete > Ease of access > High Contrast /on/)
2. Go on GitHub Desktop
3. Click on File > Options > Appearance
4. Select Dark

### Expected behavior

Working as how it should be when Windows high contrast mode is turned off.

### Actual behavior

GitHub Desktop is mostly white.

### Screenshots

When high contrast mode is turned on:
![GitHubDesktop_HighContrastMode](https://user-images.githubusercontent.com/98926797/152961642-bacbca1e-264d-440b-8358-8806d0863184.png)

When high contrast mode is turned off:
![GitHubDesktop_HighContrastModeOff](https://user-images.githubusercontent.com/98926797/152962125-d7cdde02-33db-465a-b191-a8d9e870866e.png)

**Comment by niik** [comment]
> No screenshots needed.

Hey @raymond-ntt, thanks for reaching out. Could you please provide a screenshot of what GitHub Desktop looks like when in High contrast mode. It would help us get a sense of what's going on before we spin up a Windows machine for further debugging.

**Comment by raymond-ntt** [comment]
@niik Updated with screenshots of when high contrast mode is turned on/off.

**Comment by raymond-ntt** [comment]
Issue seems to be resolved with the latest Windows update. Hence, I suspect it to be a Windows issue and not a GitHub Desktop issue. I have no more issue and I will proceed to close this ticket.
